ssn/rdf/ontology/core/ssn-common.ttl

```diff
@@ -54,6 +55,10 @@ sosa:hasProperty
 sosa:hasFeatureOfInterest 
   rdfs:isDefinedBy sosa-common: .
 
+sosa:invForProperty
```

Comment:
I don't think this is incorrect. However, wouldn't it br more accurate to have this property defined in the [`sosa` module](http://www.w3.org/ns/sosa/)?

Reply:
We use `rdfs:isDefinedBy` to point to the module where the term is introduced. 
`sosa:invForProperty` is introduced in `ssn-common`, and does not appear in any of its dependencies.